project/Build.scala

```diff
@@ -1049,6 +1049,16 @@ object Build {
       ) ++ (
           scalaJSExternalCompileSettings
       ) ++ inConfig(Compile)(Seq(
+
+         unmanagedSourceDirectories += {
+            val dir = sourceDirectory.value
+            val scalaV = scalaVersion.value
+            CrossVersion.partialVersion(scalaV) match {
+              case Some((2, 13)) if scalaV != "2.13.0-M3" => dir / "scala-2.13"
+              case _ => dir / "scala-2.10_2.13.0-M3"
+            }
```

Comment:
I had to create a source directory for Scala versions between 2.10 and 2.13.0-M3, and another for Scala 2.13.0-M4.